### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by pbayfiel-ford** [evansiroky/timezone-boundary-builder#31]
I was hoping to build this into a Gradle plugin to automate downloading timezone data into an Android app (along with another library to convert to Java) but ran into an error running the script per instructions in readme:

    getting data for Argentina
    downloading from overpass
    waiting 4 seconds
    Success, decreasing overpass request gap
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    combining border
    error converting geojson to geometry
    error converting overpass result to geojson
    n {}
    
    /Users/phil/.../timezone-boundary-builder/index.js:210
                throw e
                ^
    [object Object]

**Comment by evansiroky** [comment]
I just tried this out locally and it worked.  The error you saw was likely the result of some kind of invalid geometry that occurred in OpenStreetMap that this library couldn't process, however it must have already been fixed in OpenStreetMap.  FYI, running the whole timezone boundary building script should be considered unstable because these errors happen frequently.  I'd recommend instead making your gradle plugin download a file from the latest release instead.
